apps/davi/src/Modules/Guilds/pages/SchemeSelection/SchemeSelection.tsx

```diff
@@ -0,0 +1,204 @@
+import { useEffect, useState } from 'react';
+import { useNavigate, useSearchParams } from 'react-router-dom';
+import { FaChevronLeft } from 'react-icons/fa';
+import { useTranslation } from 'react-i18next';
+import { useNetwork } from 'wagmi';
+import { useQuery } from '@apollo/client';
+
+import { getSchemesDocument, getSchemesQuery } from '.graphclient';
+import { getApolloClient } from 'clients/apollo';
+import { useHookStoreProvider } from 'stores';
+import { SupportedSubgraph } from 'stores/types';
+
+import {
+  SidebarCard,
+  SidebarCardContent,
+  SidebarCardHeaderSpaced,
+} from 'components/SidebarCard';
+import { Divider } from 'components/Divider';
+import { Result, ResultState } from 'components/Result';
+import { IconButton } from 'components/primitives/Button';
+import { RadioInput } from 'components/primitives/Forms/RadioInput';
+import { Box } from 'components/primitives/Layout';
+import { StyledLink } from 'components/primitives/Links';
+import { Loading } from 'components/primitives/Loading';
+import { Heading } from 'components/primitives/Typography';
+import { useTypedParams } from 'Modules/Guilds/Hooks/useTypedParams';
+import { PageContainer, SidebarContent } from 'Modules/Guilds/styles';
+
+import {
+  CardContainer,
+  CardTitle,
+  NextButton,
+  RadioInputContainer,
+  StyledDivider,
+} from './SchemeSelection.styled';
+import { SchemeInfo } from './SchemeInfo';
+import { Treasury } from '../Treasury';
+import { PermissionsPage } from '../Permissions';
+
+const SchemeSelection = () => {
+  const { t } = useTranslation();
+  const { chain } = useNetwork();
+  const { guildId: daoId, chainName } = useTypedParams();
+  const navigate = useNavigate();
+  const [searchParams] = useSearchParams();
+  const discussionId = searchParams.get('ref');
+  const subdaoId = searchParams.get('subdao');
+
+  const { name: governanceName } = useHookStoreProvider();
+
+  const {
+    data,
+    loading: isSchemeLoading,
+    error: errorFetchingScheme,
+  } = useQuery<getSchemesQuery>(getSchemesDocument, {

```

Comment:
Should we consider moving this into our fetchers store? I know it seems pointless now as its a specific page for Gov 1.5 but as we add new versions of the implementation like 2.0 it might be useful having a separation from the component and the data fetching.
We could just make the hook in fetcher's interface be optional and document that is is specifically for DAOs with scheme capabilities.

Reply:
Makes sense!

I'll go with making an empty implementation for guilds to both make it explicit that this method isn't supported there and to keep the same logic we followed with other non-supported methods (like lock tokens on REP guilds).